README.md

```diff
@@ -0,0 +1,186 @@
+# Book vehicle rental frontend
```

Comment:
- [ ] Good job implementing the changes in your backend repo.

However, this PR (the frontend), has had no changes since the last review.

Kindly, implement the changes which are:

1. Correct the mobile homepage styling.
2. Make your functions ES6.

That way, your project will be ready for prime time. 💪

Reply:
Hello @Meltrust,
- Mobile homepage resposive :heavy_check_mark: 
- Covert all syntax to ES6 :heavy_check_mark: